### DISCUSSION TITLE
Support for WalletConnect 2.0

### DISCUSSION THREAD

**Discussion by Steveantor** [family/connectkit#177] [Ideas]
I believe it's about time to migrate to **WalletConnect 2.0** since v1 is being deprecated pretty sooner.
Better preparing ourselves for some minor breaking changes than being late,
since most wallets and dapps have already migrated to v2

#161 looks like added support for wagmi 12x
I do expect full support and release for v2.

**Comment by lochie** [comment]
our next minor update should include support for v2, we're just ironing out some changes that break our UX

**Reply by Steveantor** [reply]
Thanks for the update!
